src/web/app/pages-student/student-profile-page/student-profile-page.component.ts

```diff
@@ -24,7 +25,10 @@ interface StudentProfile {
   pictureKey: string;
 }
 
-interface StudentDetails {
+/**
+ * Represents detailed data for student profile.
+ */
+export interface StudentDetails {
```

Comment:
Why we need export?

Reply:
because it is also used in `student-profile-service.ts`